ktlint-core/src/main/kotlin/com/pinterest/ktlint/core/KtLint.kt

```diff
@@ -254,6 +257,10 @@ object KtLint {
         }
     }
 
+    private fun filterDisabledRules(rootNode: ASTNode, fqRuleId: String): Boolean {
+        return rootNode.getUserData(DISABLED_RULES)?.contains(fqRuleId) == false
```

Comment:
Note: this will currently work only with the standard ruleset. If you have an experimental one (or any other custom ruleset), you have to specify the rule with the prefix inside `.ktlintignore` (as you can see it's being added to the rule id at the line 207). So something like `experimental:indent` - I guess this should be mentioned in the docs or something :) Or we could change this to `any { fqRuleId.contains(it) }`, if we want to be independent of the ruleset (as I assume rule ids most of the time are unique)

Reply:
That's correct. I'd rather require fully namespaced rules than unexpectedly disabling a rule with a name collision. We would definitely update the docs if we move forward with this.